Image build on docker system with CUSTOM_LLVM=false fails

```
Step 12/27 : FROM base AS llvm-local-true
 ---> b1752060328a
Step 13/27 : COPY --from=llvm-build /llvm-project/ /llvm-project/
COPY failed: stat llvm-project/: file does not exist
make: *** [Makefile:53: triton-image] Error 1
```

This is because when CUSTOM_LLVM is false, it skips cloning the llvm repo and so the llvm-project directory never gets created.

--- a/Dockerfile.triton
+++ b/Dockerfile.triton
@@ -33,6 +33,7 @@ RUN if [ "$CUSTOM_LLVM" = "true" ]; then \
         ninja; \
     else \
         echo "Skipping LLVM build because CUSTOM_LLVM is not true"; \
+        mkdir llvm-project; \
     fi
 
 FROM registry.access.redhat.com/ubi9/python-312 AS base

--- a/Dockerfile.triton-amd
+++ b/Dockerfile.triton-amd
@@ -35,6 +35,7 @@ RUN if [ "$CUSTOM_LLVM" = "true" ]; then \
         dnf clean all && rm -rf /var/cache/dnf /tmp/*; \
     else \
         echo "Skipping LLVM build because CUSTOM_LLVM is not true"; \
+        mkdir llvm-project; \
     fi
 
 # Base Stage with ROCm Installation

--- a/Dockerfile.triton-cpu
+++ b/Dockerfile.triton-cpu
@@ -34,6 +34,7 @@ RUN if [ "$CUSTOM_LLVM" = "true" ]; then \
         ninja; \
     else \
         echo "Skipping LLVM build because CUSTOM_LLVM is not true"; \
+        mkdir llvm-project; \
     fi
 
 FROM registry.access.redhat.com/ubi9/python-312 AS base

